schemas/log.yml

```diff
@@ -156,3 +156,50 @@
 
         According to RFCs 5424 and 3164, the priority is 8 * facility + severity.
         This number is therefore expected to contain a value between 0 and 191.
+
+    - name: syslog.version
+      level: extended
+      type: long
+      example: 1
+      short: Syslog protocol version.
+      description: >
+        The version of the Syslog protocol specification. Only applicable for RFC 5424 messages.
+
+    - name: syslog.hostname
+      level: extended
+      type: keyword
+      example: example-host
+      short: The host that originated the Syslog message.
+      description: >
+        The hostname, FQDN, or IP of the machine that originally sent the Syslog message.
+
+    - name: syslog.appname
+      level: extended
+      type: keyword
+      example: sshd
+      short: The device or application that originated the Syslog message.
+      description: >
+        The device or application that originated the Syslog message, if available.
+
+    - name: syslog.procid
+      level: extended
+      type: keyword
+      example: 12345
+      short: The process name or ID that originated the Syslog message.
+      description: >
+        The process name or ID that originated the Syslog message, if available.
+
+    - name: syslog.msgid
+      level: extended
+      type: keyword
+      example: ID47
+      short: An identifier for the type of Syslog message.
+      description: >
+        An identifier for the type of Syslog message, if available. Only applicable for RFC 5424 messages.
+
+    - name: syslog.data
```

Comment:
Can we rename to `syslog.structured_data`?

I worry some users might confuse `data` with the free-form message portion of syslog events.